src/components/Summary/PersonalDataForm.tsx

```diff
@@ -13,77 +13,114 @@ const PersonalDataForm = ({
   return (
     <FormItem>
       <FormContainer>
-        <VFlexBox>
+        <VFlexBox
+          className={
+            formik.touched.firstName && formik.errors.firstName
+              ? "error-message"
+              : ""
+          }
+        >
```

Comment:
When you're done with the main app, we're gonna refactor it a little bit, Ima show you a couple of cool tricks with styled components